changes/14464-add-kolide-tables

```diff
@@ -0,0 +1,29 @@
+- added tables to the fleetd extension: 
+  - airport_util
+  - app_icons
+  - apple_silicon_security_policy
+  - falconctl_options
+  - falcon_kernel_check
+  - cryptoinfo
+  - cryptsetup_status
+  - parse_plist
+  - parse_json
+  - parse_jsonl
+  - parse_xml
+  - parse_ini
+  - dev_table_tooling
```

Comment:
```suggestion
  - dev_table_tooling
```

This table is a dependency for other tables right?

Are we adding any of the tables that depend on it? If not, let's cut this.

Reply:
no dependencies on other tables